tools/V4-CSDL-to-OpenAPI.xsl

```diff
@@ -1416,6 +1416,23 @@
 
     <xsl:if test="$direct and ($baseproperties!='' or $hereproperties!='')">
       <xsl:text>,"properties":{</xsl:text>
+      <!-- odata entity metadata properties -->
+      <xsl:if test="not($suffix) and local-name($structuredType)='EntityType'">
+        <xsl:choose>
+          <xsl:when test="$odata-version='2.0'">
+            <xsl:text>"__metadata":{
+                "type":"object",
+                "readOnly":true,
+                "description":"This name/value pair is not data, but instead, specifies the metadata for the EntityType instance that the JSON object represents.",
+                "properties":{"id":{"type":"string"},"uri":{"type":"string"},"type":{"type":"string"},"etag":{"type":"string","example":"W/\"time(123456789)\""}}
+              },
+            </xsl:text>
+          </xsl:when>
+          <xsl:otherwise>
+            <xsl:text>"@odata.id":{"type":"string","readOnly":true},"@odata.etag":{"type":"string","readOnly":true,"example":"W/\"08D7D40891852C76\""},</xsl:text>

```

Comment:
I was unable to find any evidence that these properties have changed in v4.01, though I would expect the to also have ditched the `odata` prefix and just be `@id`, `@etag`.
Please confirm.

Reply:
Yes, no infix `odata.` in V4.01

Reply:
The easiest way to find that out is by looking at the table of contents of https://docs.oasis-open.org/odata/odata-json-format/v4.01/odata-json-format-v4.01.html, subsections 4.5.x 😄